packages/main/src/themes/Table.css

```diff
@@ -54,6 +54,6 @@
     z-index: 2;
 }
 
-#loading[_is-busy] {
-	background-color: var(--_ui5_busy_indicator_block_layer);
+:host([loading]) #table ::slotted(*) {

```

Comment:
the loading attribute sets immediately but the loading animation is shown after `loadingDelay` ms. If I remember it correctly `_is-busy` is set when the animation has started so I guess you need a `has` selector.